benchmarks/dynamo/genai_layers/kernels.py

```diff
@@ -447,7 +447,6 @@ def get_shapes(self) -> tuple[tuple[int, ...], ...]:
             (32768, 4096),
             (32768, 8192),
             (32768, 16384),
-            (32768, 32768),
```

Comment:
Remove this one since such large reduction dimension for rmsnorm is very unrepresentative.